ghost/core/core/server/api/endpoints/members.js

```diff
@@ -143,6 +143,31 @@ module.exports = {
         }
     },
 
+    logout: {
+        statusCode: 204,
+        headers: {
+            cacheInvalidate: false
+        },
+        options: [
+            'id'
+        ],
+        validation: {
+            options: {
+                id: {
+                    required: true
+                }
+            }
+        },
+        permissions: {
+            method: 'destroy'
```

Comment:
Is this deffo the permission we want to use here?

I would expect this to be `'edit'` maybe? Though I'll be honest - that might be essentially the same as `'destroy'`

Not a big deal, but just wanted to make sure we don't want to change it, or add a new permission!

Reply:
I chose the most restrictive permissions (though they are indeed the same), but edit also makes sense for me. I'm not a big fan to add a new permission specifically for this